Dockerfile

```diff
@@ -24,10 +26,10 @@ RUN apk add --no-cache \
     && pip3 install --no-cache-dir --requirement /tmp/requirements.txt \
     && rm -rf /root/.cache
 
-COPY --from=devkit /usr/local/bin/goss /usr/local/bin/
-COPY --from=devkit /usr/local/bin/mindthegap /usr/local/bin/
-COPY --from=devkit /usr/local/bin/packer /usr/local/bin/
-COPY --from=devkit /usr/local/bin/packer-provisioner-goss /usr/local/bin/
+ARG BUILDARCH
+COPY --from=devkit /usr/local/bin/goss-amd64 /usr/local/bin/goss
+COPY --from=devkit /usr/local/bin/packer-${BUILDARCH} /usr/local/bin/packer
+COPY --from=devkit /usr/local/bin/packer-provisioner-goss-${BUILDARCH} /usr/local/bin/packer-provisioner-goss
```

Comment:
BUILDARCH in this case is either amd64 or arm64 isn't it the same thing?

Reply:
Depends where the binary is running - if it's on the machine where the image is going to run then it will need to be `TARGETARCH`, if it's going to run on the provisioned cluster machines, then it will need to always be `amd64` (until we support arm64 clusters of course...).

Reply:
This is where i got stuck with this PR - because I don't know where the copied binaries are actually going to run.